datascience/tables.py

```diff
@@ -115,11 +129,52 @@ def from_columns_dict(cls, columns):
 
     @classmethod
     def read_table(cls, filepath_or_buffer, *args, **vargs):
-        """Read a table from a file or web address.
+        """Read a table from a file or web address."""
```

Comment:
What's the purpose/effect of changing this line?

Reply:
There's no effect. Misinterpreted a previous PR, my bad!

Reply:
I've fixed the extra quotes in the last commit.